entity-framework/ef6/fundamentals/configuring/code-based.md

```diff
@@ -13,20 +13,20 @@ Configuration for an Entity Framework application can be specified in a config f
 
 Configuration in a config file is described in a [separate article](xref:ef6/fundamentals/configuring/config-file). The config file takes precedence over code-based configuration. In other words, if a configuration option is set in both code and in the config file, then the setting in the config file is used.  
 
-## Using DbConfiguration  
+## Using `DbConfiguration`
 
-Code-based configuration in EF6 and above is achieved by creating a subclass of System.Data.Entity.Config.DbConfiguration. The following guidelines should be followed when subclassing DbConfiguration:  
+Code-based configuration in EF6 and above is achieved by creating a subclass of System.Data.Entity.Config.`DbConfiguration`. The following guidelines should be followed when subclassing `DbConfiguration`:  
```

Comment:
The first occurrence is odd

Reply:
Thanks, fixed up in 3395d72675a40d025afff773e44d749e0270333c.